lib/router/index.js

```diff
@@ -44,7 +44,7 @@ function Router(options) {
   // TODO:
   // instead, use: https://www.npmjs.org/package/path-to-regexp
   // (or: https://www.npmjs.org/package/path-match)
-  this._privateRouter = express();
+  this._privateRouter = express.Router();
```

Comment:
_/me wondering if we can assign a whole `Router` to be the "controller" at a wildcard path... opens up all kinds of possibilities like a) versioning, b) phasing out the sails controllers altogether._